tests/python/test_nonkeyed_cmd.py

```diff
@@ -61,6 +61,31 @@ def test_dbcluster_info_command(use_cluster):
         with pytest.raises(RedisReplyError):
             client.get_db_cluster_info(address)
 
+    # get env var to set through client init
+    ssdb = os.environ["SSDB"]
+    address = [ssdb]
+    del os.environ["SSDB"]
+
+    # Init client
+    client = Client(address=ssdb, cluster=use_cluster)
+
+    # Get a mock model
+    model = mock_model.create_torch_cnn()
+
+    # set the mock model
+    client.set_model("ai_info_cnn", model, "TORCH", "CPU")
+
+    # Check with valid address and model key
+    ai_info = client.get_ai_info(address, "ai_info_cnn")
+    assert len(ai_info) != 0
+
+    # Check that invalid address throws error
+    with pytest.raises(RedisReplyError):
+        client.get_ai_info(["no_host:6379"], "ai_info_cnn")
+
```

Comment:
Should be checking here for RedisRuntimeError

Reply:
In ``client.py`` I am seeing that all ``RuntimeError`` are being reraised as ``RedisReplyError``.  That seems to be the current standard.  I'm also not seeing a ``RedisRuntimeError`` in ``error.py`` -- all current python exceptions seem to  inherit from ``RuntimeError``

Reply:
That changes with the Python Error Handling PR --- sorry, should have been more clear

Reply:
Ah, makes more sense.  Thanks!